model/distro/distro.go

```diff
@@ -274,6 +284,18 @@ func (d *Distro) MaxDurationPerHost() time.Duration {
 	return evergreen.MaxDurationPerDistroHost
 }
 
+// IsPowerShellSetup returns whether or not the setup script is a powershell
+// script.
+func (d *Distro) IsPowerShellSetup() bool {
+	script := bytes.NewBufferString(d.Setup)
+	header, err := script.ReadString('\n')
```

Comment:
Does this break if the first line in the setup script is a newline?

Reply:
Yeah it will. I didn't see any scripts starting with a newline, but I changed the implementation to look for the shebang (`#!`) line instead.